src/igvfd/types/file.py

```diff
@@ -644,6 +644,84 @@ def summary(self, request, content_summary, file_set, assembly=None, transcripto
              if x is not None]
         )
 
+    @calculated_property(
+        define=True,
```

Comment:
can you explain to me how define=True is working here?

Reply:
Based on what I understand, including define=True will make sure this property is calculated in advance, therefore it could be used by the other calculated properties. Another place where it is used is the `content_summary` calculated property on Matrix File.

Reply:
got it thanks